contracts/staking/slashing/ISlashingManager.sol

```diff
@@ -0,0 +1,15 @@
+pragma solidity ^0.5.2;
+import {StakingInfo} from "../StakingInfo.sol";
+import {Registry} from "../../common/Registry.sol";
+
+
+contract ISlashingManager {
```

Comment:
I don't think there is a need to initialize storage variables to a specific value here.
Might be a good idea to keep storage and interface separate.

Reply:
done